### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by hannasavka**
- [DEMO LINK](https://hannasavka.github.io/layout_catalog/)
- [TEST REPORT LINK](https://hannasavka.github.io/layout_catalog/report/html_report/)

- [ ] All component follow BEM and use SCSS
- [ ] repaeted sizes and special colors are put to variables
- [ ] Grid is used for the columns
- [ ] cards are shown in 1, 2, 3 or 4 columns based on screen resolution
- [ ] All changes on `:hover` are smooth
- [ ] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your styles don't working, we communicate with this in the fe_chat
image

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,468 @@
...
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
```
Comment (volodymyr-soltys97) [inline src/index.html:25]: Maybe you need classes to the tag li?
```suggestion
          <li class="nav__item">
```

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,468 @@
...
+            <a class="nav__item is-active" href="#apple">apple</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:26]: Fix it everywhere
```suggestion
            <a class="nav__link is-active" href="#apple">apple</a>
```

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,468 @@
...
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card" data-qa="card">
```
Comment (volodymyr-soltys97) [inline src/index.html:117]: Consider will be better it you use the article tag here for the card
```suggestion
      <article class="card" data-qa="card">
```

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,120 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  perspective: 400px;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:7]:
```suggestion
```

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:45]: Default value, so you can just remove it everywhere
```suggestion
```

File: src/styles/utils/variables.scss [hunk 6]
```diff
@@ -0,0 +1,6 @@
+$background-color-card: #FFF;
+$background-color-header: #FFF;
+$background-button-color: #00ACDC;
+$color-header: #00ACDC;
```
Comment (volodymyr-soltys97) [inline src/styles/utils/variables.scss:4]: Why did you create two variables with the same values?
```suggestion
$background-color-card: #FFF;
$color-blue: #00ACDC;
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Comment by hannasavka** [comment]
.

**Comment by hannasavka** [comment]
.

**Pull request closed without merging**